.github/workflows/internal-release.yml

```
@@ -72,6 +72,8 @@ jobs:
       RELEASE_TYPE: 'PLAY'
       BUILD_TYPE: 'appbundle'
       FLAVOR: 'main_google_play'
+      SCANNER: 'google_play'
+      APP_REVIEW: 'uri_store'
```

```suggestion
      SCANNER: 'mlkit'
      APP_REVIEW: 'google_play'
```

It's released to the playstore